HerPublicWebsite/Views/StaticPages/SessionExpired.cshtml

```diff
@@ -0,0 +1,20 @@
+@using HerPublicWebsite.Controllers
+@{
+    ViewBag.Title = "Session expired";
+}
+
+<div class="govuk-grid-row">
```

Comment:
I assume this design came from https://design-system.dwp.gov.uk/patterns/manage-a-session-timeout
As it's not a standard Gov.UK design system pattern can we include a link in a comment please?

Reply:
Kay thinks the DWP wording is preferable as it's non-technical, so will stick with that for now. Will be easy to update later if we wish.